.github/workflows/update_protos.yml

```diff
@@ -34,6 +34,9 @@ jobs:
       - name: Format
         run: make format
 
+      - name: Update Proto Tag
+        run: sed -i -e "s/apiTag = 'v[0-9]*\.[0-9]*\.[0-9]*'/api_tag = '${{ github.event.client_payload.tag }}'" > lib/src/utils.dart
+
```

Comment:
Can we update the proto tag earlier, and then update the Makefile so that it reads the `lib/src/utils.dart` file and grabs the `apiTag`, and then generates that specific api tag?